x-pack/plugins/infra/public/components/metrics_explorer/chart_context_menu.test.tsx

```diff
@@ -82,4 +93,41 @@ describe('MetricsExplorerChartContextMenu', () => {
     expect(menuItems.length).toBe(1);
     expect(menuItems.at(0).prop('disabled')).toBeTruthy();
   });
+
+  it('should not display "Open in Visualize" when unavailble in uiCapabilities', async () => {
```

Comment:
I can see that become confusing quickly when more menu items are added.

How about we either name the test case in a way that it reflects the actual check or we generalize it to actually check the absence of the visualize link regardless of the other menu entries?

Reply:
coming with #37136